### PR TITLE
Follow-up edits to PR#2553

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up edits to https://github.com/openshift/openshift-docs/pull/2553

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tnguyen-rh @adellape @bfallonf @tpoitras please peer review

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -1583,22 +1587,27 @@ env: <1>
 
 ----
 
-<1> Similar to the link:#buildconfig-environment[*BuildConfig* environment]
+<1> Similar to the link:#buildconfig-environment[`*BuildConfig*` environment]
```
Comment (tnguyen-rh) [inline dev_guide/builds.adoc]: Need `xref:buildconfig-environment` here.

**Comment by tnguyen-rh** [comment]
@ahardin-rh One small fix required, otherwise LGTM.

**Comment by bfallonf** [comment]
@ahardin-rh LGTM as well!

File: dev_guide/builds.adoc (hunk 1 above)
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: @tnguyen-rh Nice catch! Thanks!

**Comment by ahardin-rh** [comment]
@tnguyen-rh @bfallonf Thanks for the eyeballs! :eyes:

**Comment by ahardin-rh** [comment]
Revision history in original PR

**Pull request merged**